torch/_utils.py

```diff
@@ -911,7 +911,9 @@ def _functionalize_sync(t):
 
 
 @functools.lru_cache(2)
-def _get_device_module(device_type: str):
+def _get_device_module(device_type: str = None):
```

Comment:
```suggestion
def _get_device_module(device_type: Optional[str] = None):
```
Implicit optionals should not be used.

Reply:
Thanks, got you. Change done.